src/core/tests/ov_tensor_test.cpp

```diff
@@ -709,6 +709,78 @@ TEST_F(OVTensorTest, readRangeRoiBlobStringTensor) {
     }
 }
 
+TEST_F(OVTensorTest, checkIsContinuousTensor2Dimensions) {
+    ov::Tensor tensor(ov::element::f32, ov::Shape{32, 128});
+    auto data = tensor.data();
+    auto strides = tensor.get_strides();
+
+    ov::Tensor view_tensor0(ov::element::f32, ov::Shape{16, 128}, data, strides);
+    EXPECT_EQ(view_tensor0.is_continuous(), true);
+
+    ov::Tensor view_tensor1(ov::element::f32, ov::Shape{1, 128}, data, strides);
+    EXPECT_EQ(view_tensor1.is_continuous(), true);
+
+    ov::Tensor view_tensor2(ov::element::f32, ov::Shape{1, 16}, data, strides);
+    EXPECT_EQ(view_tensor2.is_continuous(), true);
+
+    ov::Tensor view_tensor3(ov::element::f32, ov::Shape{2, 16}, data, strides);
+    EXPECT_EQ(view_tensor3.is_continuous(), false);
+}
+
+TEST_F(OVTensorTest, checkIsContinuousTensor3Dimensions) {
+    ov::Tensor tensor(ov::element::f32, ov::Shape{5, 32, 128});
+    auto data = tensor.data();
+    auto strides = tensor.get_strides();
+
+    ov::Tensor view_tensor0(ov::element::f32, ov::Shape{2, 32, 128}, data, strides);
+    EXPECT_EQ(view_tensor0.is_continuous(), true);
```

Comment:
Optional these test could be parametrized test or use local scope:
```suggestion
    {
        ov::Tensor view_tensor(ov::element::f32, ov::Shape{2, 32, 128}, data, strides);
        EXPECT_EQ(view_tensor.is_continuous(), true);
    }
```
Then local tensor view doesn't have to be with index

Reply:
I used only one view tensor tight now. Can we skip the parametrized test for now?